content/pt-br/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Obtendo uma Imagem de um Registro Privado
+content_type: task
+weight: 100
+---
+
+<!-- overview -->
+
+Esta página mostra como criar um Pod que usa uma 
+{{< glossary_tooltip text="Secret" term_id="secret" >}} para puxar uma imagem
+de um registro de imagens de contêiner ou de um repositório privado. Existem muitos registros privados em uso. 
+Esta tarefa usa o [Docker Hub](https://www.docker.com/products/docker-hub)
+como exemplo de registro.
+
+{{% thirdparty-content single="true" %}}
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}}
+
+* Para fazer este exercício, você precisa da ferramenta de linha de comando `docker`, e um
+  [Docker ID](https://docs.docker.com/docker-id/) para o qual você conheça a senha.
+* Se você estiver usando um registro de contêiner privado diferente, você precisa da 
+ferramenta de linha de comando para esse registro e suas informações de login. 
+
+<!-- steps -->
+
+## Logar em Docker Hub
+
+Em seu laptop, você deve autenticar-se em um registro para puxar uma imagem particular.
+
+Use a ferramenta `docker` para logar-se no Docker Hub. Veja a seção _log in_ do
+[Contas do Docker ID](https://docs.docker.com/docker-id/#log-in) para mais informações.
+
+```shell
+docker login
+```
+
+Quando solicitado, insira o seu Docker ID, e então a credencial que você deseja usar (token de acesso,
+ou a senha para o seu Docker ID).
+
+O processo de login cria ou atualiza um arquivo `config.json` que contém um token de autorização. Analise [como o Kubernetes interpreta esse arquivo](/docs/concepts/containers/images#config-json). 
+
+Veja o arquivo `config.json`:
+
+```shell
+cat ~/.docker/config.json
+```
+
+A saída contém uma seção semelhante a esta:
+
+```json
+{
+    "auths": {
+        "https://index.docker.io/v1/": {
+            "auth": "c3R...zE2"
+        }
+    }
+}
+```
+
+{{< note >}}
+Se você usar o armazenador de credenciais do Docker, você não vai ver essa entrada `auth`, mas sim uma `credsStore` com o nome do armazenador como valor.
+{{< /note >}}
+
+## Crie um `Secret` com base nas credenciais existentes {#registry-secret-existing-credentials}
+
+Um cluster de Kubernetes usa o `Secret` do tipo `kubernetes.io/dockerconfigjson` para autenticar com um registro de contêineres
+para pegar uma imagem privada.
+
+Se você já executou `docker login`, você pode copiar
+essa credencial para o Kubernetes:
+
+```shell
+kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Se você precisar de mais controle (por exemplo, para definir um namespace ou um rótulo no novo
+segredo) então você pode personalizar o segredo antes de armazená-lo.
+Tenha certeza de:
+
+- definir o nome do item de dados como `.dockerconfigjson`
```

Comment:
```suggestion
- definir o nome do campo como `.dockerconfigjson`
```
Suggestion for readability. This reads better in Portuguese and we have an example that follows.